hack/testing/logging.sh

```diff
@@ -36,6 +36,31 @@ USE_LOCAL_SOURCE=${USE_LOCAL_SOURCE:-false}
 TEST_PERF=${TEST_PERF:-false}
 ES_VOLUME=${ES_VOLUME:-/var/lib/es}
 ES_OPS_VOLUME=${ES_OPS_VOLUME:-/var/lib/es-ops}
+# default CI image uses journal by default - allow to control if logging uses it or not
+USE_JOURNAL=${USE_JOURNAL:-false}
+
+if [ $USE_JOURNAL = false ] ; then
+    # see if docker is using the journal log driver - if so, change it to json-file
+    if grep -- '--log-driver=journald' /etc/sysconfig/docker > /dev/null 2>&1 ; then
```

Comment:
I suggest using `grep -q` instead of blackholing output - then you shouldn't have any output unless something is really wrong.